frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizations.tsx

```diff
@@ -173,9 +212,9 @@ const PendingOrganizations = ({
     );
     return orgsSortedByNewest.map((o) => (
       <tr key={o.externalId} className="sr-org-row">
-        <td>{o.name}</td>
-        <td>{`${o.adminFirstName} ${o.adminLastName}`}</td>
-        <td>
+        <td className="word-break-break-word">{o.name}</td>
```

Comment:
⚠️ adding this fixes the following issue:

**Before**
<img width="1749" alt="Screenshot 2023-09-07 at 21 11 48" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/8480b830-3e56-4137-983a-8176711fffdb">


**After**
<img width="1750" alt="Screenshot 2023-09-07 at 21 12 11" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/3cccc21c-a20f-4c1f-a822-efed98ba3cb5">

Reply:
Thanks for fixing that! It has bug me for a while xD